sources/packages/web/src/views/supporting-user/SupportingInformation.vue

```diff
@@ -80,7 +123,7 @@ export default {
       required: true,
     },
   },
-  setup(props: any) {
+  setup(props: any, context: SetupContext) {
```

Comment:
Please use the `export default defineCompoent` to avoid declaring the `props` as any. The type for `context` will be inferred also.

Reply:
In order for the context also get the proper type we need also declare the events, for instance, `emits: ["myEvent"]`. That said, there are comments asking if there is a reason for those events to be raised.
Please check if the events are needed and add them to `emits` or remove them and remove the `context` also, please.

Reply:
I'm not emitting anymore, thanks!